crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1784,7 +1927,7 @@ mod tests {
         OwningPtr::make(value, |ptr| {
             // SAFETY: value is valid for the component layout
             unsafe {
-                world.insert_resource_by_id(component_id, ptr);
+                world.insert_resource_by_id(component_id, true, ptr);
```

Comment:
need a safety for why `true` is correct

Reply:
Added a note about how R is Send.

Reply:
Left a comment that the array is `Send`.